src/Couchbase.Lite.Support.Android/Activate.cs

```diff
@@ -56,7 +56,7 @@ public static void Activate(Context context)
             Context = context ?? throw new ArgumentNullException(nameof(context));
         }
 
-        internal static void CheckVersion()
+        public static void CheckVersion()
```

Comment:
This is not the way to handle this, did you use the `InternalsVisibleTo` attribute?